app/helpers/users_helper.rb

```diff
@@ -1,6 +1,6 @@
 module UsersHelper
   def last_login_on_column(record)
-    _("%s ago") % time_ago_in_words(record.last_login_on) if record.last_login_on
+    date_time_relative(record.last_login_on) if record.last_login_on
   end
```

Comment:
hmm, maybe we should get rid of this helper (and the condition) and just move it to the index page directly?

Reply:
Sure!